src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/health/HealthCheck.kt

```diff
@@ -13,7 +13,7 @@ abstract class HealthCheck(private val webClient: WebClient, private val timeout
       .retrieve()
       .toEntity(String::class.java)
       .block(timeout)
-    Health.up().withDetail("HttpStatus", responseEntity.statusCode).build()
+    Health.up().withDetail("HttpStatus", responseEntity!!.statusCode).build()
```

Comment:
I couldn't work out what caused this to be needed... definitely needed?

Reply:
Just a compiler warning as the block() call return is Nullable